app/models/other_income_source.rb

```diff
@@ -1,10 +1,13 @@
 class OtherIncomeSource < ApplicationRecord
   include MonthlyEquivalentCalculator
 
+  VALID_INCOME_SOURCES = %w[friends_or_family maintenance_in property_or_lodger student_loan pension].freeze
+
   belongs_to :gross_income_summary
   has_many :other_income_payments
 
   validates :gross_income_summary_id, :name, presence: true
+  validates :name, inclusion: { in: VALID_INCOME_SOURCES }
```

Comment:
A suggestion of adding a custom error message as the default will be "is not included in the list"
Is there a test for this, if needed?

`validates : name, inclusion: { in: VALID_INCOME_SOURCES, message: "%{value} is not a valid income source" }`

Reply:
Both good suggestions - I only saw this message after I merged it.  best to request changes.  I'll include it in the next ticket.